### PR TITLE
Fixed footer layout

### LINKED ISSUE #12549: Footer layout gets messed up in mobile view
[//]: # (Before logging this issue, please post to the Report a Bug guild from the Habitica website's Help menu. Most bugs can be handled quickly there. If a GitHub issue is needed, you will be advised of that by a moderator or staff member -- a player with a dark blue or purple name. It is recommended that you don't create a new issue unless advised to.)

[//]: # (Bugs in the mobile apps can also be reported there.)

[//]: # (If you have a feature request, use "Help > Request a Feature", not GitHub or the Report a Bug guild.)

[//]: # (For more guidelines see https://github.com/HabitRPG/habitica/issues/2760)

[//]: # (Fill out relevant information - UUID is found from the Habitia website at User Icon > Settings > API)
### General Info
  * UUID: af292e10-b7c0-4a9b-866a-af13868920ba
  * Browser: Google Chrome
  * OS: Windows

### Description
[//]: # (Describe bug in detail here. Include screenshots if helpful.)
Footer Bug Mobile View

Footer Layout becomes messy in mobile view and donate button touches upper paragraph. Privacy policy and terms and conditions gets messed up due to their alignment. This issue can be fixed by changing their alignment to center.  
Sir, please assign me this issue i will fix it asap.

#### Console Errors
[//]: # (Include any JavaScript console errors here.)

### PR DESCRIPTION
**Pull request by VipulKhandelwal1999**
[//]: # (Note: See http://habitica.fandom.com/wiki/Using_Your_Local_Install_to_Modify_Habitica%27s_Website_and_API for more info)

[//]: # (Put Issue # here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes #12549 

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)
I have fixed alignment of footer links in mobile view as well as desktop view

Desktop View:-
image

Mobile View:-
image

[//]: # (Put User ID in here - found on the Habitica website at User Icon > Settings > API)

----
UUID: af292e10-b7c0-4a9b-866a-af13868920ba

### CODE REVIEW

File: website/client/src/components/appFooter.vue [hunk 1]
```diff
@@ -6,16 +6,12 @@
       <div class="row">
         <div class="col-12 col-md-2">
           <h3>
-            <a
-              href="https://itunes.apple.com/us/app/habitica/id994882113?ls=1&mt=8"
-              target="_blank"
-            >{{ $t('mobileIOS') }}</a>
+            <a href="https://itunes.apple.com/us/app/habitica/id994882113?ls=1&mt=8"
```
Comment (paglias) [inline website/client/src/components/appFooter.vue:9]: It looks like you have something like a plugin or a setting in your text editor that seems to change a lot of formatting in files edited, can you disable it?

Comment (paglias) [reply website/client/src/components/appFooter.vue:9]: It makes it really difficult to see what actually changed

**Review by paglias — COMMENTED** [review]
Fix looks good

**Comment by paglias** [comment]
thanks @VipulKhandelwal1999 , welcome to the 2nd tier

**Comment by VipulKhandelwal1999** [comment]
> thanks @VipulKhandelwal1999 , welcome to the 2nd tier

@paglias  You're welcome 🙂🙂

**Comment by Saiyara6133** [comment]
After I fetched the latest changes, which included this fix, it seems that the `Toggle Debug Menu` button disappeared when testing in the browser in development mode.

**Comment by VipulKhandelwal1999** [comment]
@Saiyara6133 I am really sorry, I just forgot to put "!" symbol in if statement, I will fix it and will send you PR ASAP

**Comment by VipulKhandelwal1999** [comment]
@Saiyara6133 I have fixed it and sent PR on issue number #12861

**Comment by Saiyara6133** [comment]
@VipulKhandelwal1999 Thank you!

**Comment by VipulKhandelwal1999** [comment]
@Saiyara6133 You're Welcome and thank you for telling me this because it was quite an important issue with my PR.

**Pull request merged**